app/Actions/Discord/Channels/CreateChannel.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Actions\Discord\Channels;
+
+use ReflectionException;
+use Saloon\Contracts\Response;
+use App\Enums\Discord\ChannelType;
+use App\Actions\Discord\DiscordAction;
+use Saloon\Exceptions\PendingRequestException;
+use Saloon\Exceptions\InvalidResponseClassException;
+use App\Http\Integrations\Discord\Channel\Requests\CreateGuildChannel;
+
+class CreateChannel
+{
+    use DiscordAction;
+
+    /**
+     * @throws InvalidResponseClassException
+     * @throws ReflectionException
+     * @throws PendingRequestException
+     */
+    public function execute(
+        string $name,
+        ChannelType $channelType = ChannelType::TEXT,
+        ?string $parentId = null
```

Comment:
What is your opinion on union types?

I think its more readable and clear using it, like null|string

Reply:
for null its ok to use "?"

Reply:
string|int|null ok
bcuz ?string|int not readable and may not work